nss-day-cohort-39/tabloid-full-stack-velours
# Post Images Upload

**Changes** 
- Changed all images on post to be uploaded images into the wwwroot folder of our C# project
- Created UploadsController to handle the get'ing and posting of images into the wwwroot folder
- Created UploadImgProvider in react to allow for fetch calls to webapi
- Updated logic on Post Listing, Add Posts, and Edit Posts to call for img upload instead of displaying hardcoded links to internet images

**Test**
1. First run the npm i axios in client directory - in npm commands in spreadsheet
2. Run a SQL query to update all ImageLocations for Posts in Db - in spreadsheet
3. Log in and go to Posts and MyPosts and confirm you see the infamous "Hack" as the picture for all posts
4. Add a post and upload your own image - confirm it updates PostList and your image is viewable
5. Edit a post and change the image currently associated with post: Also try and not edit the image when you edit a post and confirm it stays the same
